app/lib/mi_bridges/driver.rb

```diff
@@ -129,15 +129,15 @@ def complete_application
     def run_flow(flow)
       flow.each do |klass|
         begin
-          @page = klass.new(@snap_application, logger: logger)
-          @page.setup
-          @page.fill_in_required_fields
-          @page.continue
+          mi_bridges_page = klass.new(snap_application, logger: logger)
```

Comment:
I renamed these to clarify intent -- below, we use `page`, in the `page_title` method...which is actually looking to to the capybara `page` method rather than the `page` that we instantiate here, but that was not clear with our previous naming

Reply:
good catch! Thank you